src/register-command.tsx

```diff
@@ -153,6 +153,22 @@ async function getBlockTreeAndMode(maybeUUID: string) {
   return { tree, mode };
 }
 
+function getReferenceContent(content: string): string | null {
```

Comment:
Not sure if you want to also consider cases that there are multiple block references been embedded. Also, I think you can simplify the logic with a regex like this one:
`content.matchAll(/\(\(([a-zA-Z0-9-]*)\)\)/g))`, which returns all the matched block ref ids

Reply:
ok, I modify it after.